inconspiquous/dialects/qft.py

```diff
@@ -0,0 +1,40 @@
+from xdsl.dialects.builtin import IntAttr
+from xdsl.ir import Dialect
+from xdsl.irdl import irdl_attr_definition, ParameterDef
+from xdsl.parser import AttrParser
+from xdsl.printer import Printer
+
+from inconspiquous.gates import GateAttr
+
+
+@irdl_attr_definition
+class QFTAttr(GateAttr):
+    name = "qft.n"
```

Comment:
The alternative to this would be to have an `algorithm` or `algo` dialect and put `qft` inside. Do you see anything else being in this dialect?